### PR TITLE
fix estimator license

### PR DESCRIPTION
**Pull request by whitewindmills**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: hack/verify-license.sh [hunk 1]
```diff
@@ -35,12 +35,10 @@ missing_license_header_files="$($ADDLICENSE_BIN \
   -ignore "docs/**" \
   -ignore ".github/**" \
   -ignore "third_party/**" \
-  -ignore "**/pb/**" \
   -ignore "**/*.md" \
   -ignore "**/*.yaml" \
   -ignore "**/*.yml" \
   -ignore "**/*.json" \
-  -ignore "**/*.proto" \
```
Comment (RainbowMango) [inline hack/verify-license.sh:43]: Just a question, seems the `verify-license.sh` would ignore the `generated` files, but here doesn't have a flag like `-ignore *generated*.go*`, do you know why?

Comment (whitewindmills) [reply hack/verify-license.sh:43]: The script `verify-license.sh` uses [`addlicense`](https://github.com/google/addlicense/tree/master) to check licenses. It is the hard-coded default behavior.
image

Comment (RainbowMango) [reply hack/verify-license.sh:43]: Thanks for the clarification. I see it the file with `Code Generated .* DO NOT EDIT` will be treated as a generated file.
https://github.com/google/addlicense/blob/4caba19b7ed7818bb86bc4cd20411a246aa4a524/main.go#L360-L370

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve
Thanks.

**Pull request merged**